docs/references/nextjs/auth.mdx

```diff
@@ -32,54 +65,72 @@ import { auth } from '@clerk/nextjs';
 
 export async function GET() {
   const {userId, getToken} = auth();
+
   if(!userId){
     return new Response("Unauthorized", { status: 401 });
   }
+
   const token = await getToken({template: "supabase"});
```

Comment:
```suggestion
  try {
    const token = await getToken({ template: "supabase" });
  } catch (error) {
    return NextResponse.json({ error });
  }
```

Prevents it from crashing

Reply:
`getToken()` shouldn't cause a crash, just return null if there is no signed in user.

Reply:
It will if it doesn't find a template called "supabase" in my testing

Reply:
```
js:140:13) {
  toString: [Function (anonymous)],
  status: 404,
  clerkTraceId: 'b49f6ccda0afd0e1bb25fe9a879af04a',
  clerkError: true,
  errors: [
    {
      code: 'resource_not_found',
      message: 'JWT template not found',
      longMessage: 'No JWT template exists with name: supaba2se',
      meta: [Object]
    }
  ]
```

Reply:
TIL. I hadn't fetched a token from a template in a Route Handler before. On the frontend the try/catch does nothing. `null` is returned instead of an error.